static/src/stylesheets/module/content/_media.head.scss

```diff
@@ -22,3 +22,14 @@
 .media-body {
     position: relative;
 }
+
+.tonal--tone-media .content__headline {
```

Comment:
Have added a note in the file to explain. Keeps the icon proportionate to the text without having to add breakpoints.